project/src/pages/main-screen/main-screen.tsx

```diff
@@ -1,12 +1,14 @@
-import PlaceCard from '../../components/place-card/place-card';
+import Logo from '../../components/logo/logo';
+import PlaceCard from '../../components/property-card/property-card';
+import UserProfile from '../../components/user-profile/user-profile';
 
 type MainScreenProps = {
-  placeCardsCount: number;
+  propertyCardsCount: number;
 }
 
-export default function MainScreen({placeCardsCount}: MainScreenProps): JSX.Element {
-  const placeCardsList = Array.from(
-    {length: placeCardsCount},
+export default function MainScreen({propertyCardsCount}: MainScreenProps): JSX.Element {
+  const propertyCardsList = Array.from(
+    {length: propertyCardsCount},
     (v, k) => <PlaceCard key={k}/>
```

Comment:
Если элемент не нужен то общепринято обозначать его как _
```suggestion
    (_, k) => <PlaceCard key={k}/>
```